src/modules/core/components/emoji/emoji.tsx

```diff
@@ -0,0 +1,28 @@
+import { type FC } from 'react';
+// animations
+import Lottie from 'lottie-react';
+import sunglasses from '@/assets/lottie-files/emoji/emoji-sunglasses.json';
+import heartEyes from '@/assets/lottie-files/emoji/emoji-heart-eyes.json';
+
+// keys
+const emojies = {
+  sunglasses,
+  heart_eyes: heartEyes,
```

Comment:
I don't really like `heart_eyes` notation, can'e we use `heartEyes` instead?

Reply:
We have icon name - image-not-found, so suggest use same principe - heart-eyes.